firebase-sessions/src/main/kotlin/com/google/firebase/sessions/api/SessionSubscriber.kt

```diff
@@ -16,12 +16,15 @@
 
 package com.google.firebase.sessions.api
 
+import androidx.annotation.Discouraged
+
 /** [SessionSubscriber] is an interface that dependent SDKs must implement. */
 interface SessionSubscriber {
   /** [SessionSubscriber.Name]s are used for identifying subscribers. */
   enum class Name {
     CRASHLYTICS,
     PERFORMANCE,
+    @Discouraged(message = "This is for testing purposes only.") MATT_SAYS_HI,
```

Comment:
Why this?

Reply:
This is so we can make integration tests that don't rely on Crashlytics or Perf being initialized, but won't break if we do add in Crashlytics or Perf. We use it in the test app. I was worried when we do the gradle configs to include Crashlytics, include Perf, that there would be configurations that wouldn't be valid.